### PR TITLE
Redis Desktop Manager 0.9.9 (new formula)

### PR DESCRIPTION
**Pull request by jameshilliard**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/rdm.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  desc "Cross-platform GUI management tool for Redis"
+  homepage "https://redisdesktop.com/"
+  url "https://github.com/uglide/RedisDesktopManager/archive/0.9.9.tar.gz"
+  sha256 "8f7244813d5a82bc686a3af9f98e1372479c9861d724ba7e7a2b162ccf35d9bb"
+  head "https://github.com/uglide/RedisDesktopManager.git"
+
+  depends_on "cmake" => :build
+  depends_on :xcode => :build
+  depends_on :macos
+  depends_on :macos => :high_sierra
```
Comment (jameshilliard) [inline Formula/rdm.rb]: Not sure why jenkins is trying to build on sierra when I have the minimum version set to high_sierra here.

**Comment by SMillerDev** [comment]
I'm not sure how the parsing works but you have the dependency on macOS set twice. It might pick one with or without restrictions on the version.

**Comment by jameshilliard** [comment]
Yep, seems that fixed it.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Acceptable-Formulae#stuff-that-builds-an-app

Might also apply here.

**Comment by jameshilliard** [comment]
Cask is not possible since upstream does not provide any binary builds for osx. I saw there were other applications in homebrew that were packaged as .app's.

**Comment by jameshilliard** [comment]
Anything I should change?

**Comment by jameshilliard** [comment]
rebased

**Pull request closed without merging**